docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -2,6 +2,18 @@
 Query management properties
 ===========================
 
+``query.execution-policy``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-string`
+* **Default value:** ``phased``
+* **Session property:** ``execution_policy``
+
+The execution policy improves the wall time for query processing with the
+``phased`` scheduling policy. To restore previous previous policy behavior, set
```

Comment:
`To restore previous previous policy behavior`

I think such comment makes sense only in release notes. Maybe here we should briefly enumerate all policies?

Reply:
That would be great, the info I have is based on the [368 release notes](https://trino.io/docs/current/release/release-368.html). I'd love to get some clarity on the specific policies @sopel39